support/vm/invocation_context.go

```diff
@@ -316,13 +342,65 @@ func (ic *invocationContext) Send(toAddr address.Address, methodNum abi.MethodNu
 
 	newCtx := newInvocationContext(ic.rt, ic.topLevel, newMsg, fromActor, ic.emptyObject)
 	ret, code := newCtx.invoke()
+
+	ic.mergeSubStat(newCtx.toActor.Code, newMsg.method, newCtx.stats)
+
 	err := ret.Into(out)
 	if err != nil {
 		ic.Abortf(exitcode.ErrSerialization, "failed to serialize send return value into output parameter")
 	}
 	return code
 }
 
+func (ic *invocationContext) mergeSubStat(code cid.Cid, methodNum abi.MethodNum, newStats *CallStats) {
+	if ic.stats.SubStats == nil {
```

Comment:
I _think_ Go might let you implement `CallStats.MergeStats` to accept a `nil` receiver and still do this, which would be nice. Otherwise I'd prefer we init `ic.stats.SubStats` with a pointer to zeros, or use a non-pointer.